npm/ng-packs/packages/theme-shared/src/lib/components/password/password.component.ts

```diff
@@ -1,9 +1,12 @@
 import { Component, forwardRef, Injector, Input } from '@angular/core';
 import { AbstractNgModelComponent } from '@abp/ng.core';
-import { NG_VALUE_ACCESSOR } from '@angular/forms';
+import { FormsModule, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'abp-password',
+  standalone: true,
+  imports: [CommonModule, FormsModule],
```

Comment:
Don't need to use `CommonModule` use [NgClass](https://angular.io/api/common/NgClass) instead.

```ts
imports: [NgClass, FormsModule],
```

Reply:
It alread deprecated and It will removed. no need to improve